packages/editor-ui/src/constants.ts

```diff
@@ -172,6 +172,7 @@ export const NODE_CREATOR_OPEN_SOURCES: Record<
 	'': '',
 };
 export const CORE_NODES_CATEGORY = 'Core Nodes';
+export const CORE_VARIANT_NODES_CATEGORY = 'Core Variant Nodes';
```

Comment:
this has one function.. to not show description.. is it used for anything else? seems maybe it's unneeded and we should do that some other way.

Reply:
Removed it